nativelib/src/main/resources/gc/immix/Constants.h

```diff
@@ -49,6 +49,8 @@
 
 #define LARGE_BLOCK_MASK (~((1 << LARGE_OBJECT_MIN_SIZE_BITS) - 1))
 
-#define GROWTH_RATE 30
+#define EARLY_GROWTH_THRESHOLD (128 * 1024 * 1024UL)
+#define EARLY_GROWTH_RATE 2.0
+#define GROWTH_RATE 1.414213562
```

Comment:
Is there a reason for using sqrt(2) and not 1.4 ?

Reply:
sqrt(2) makes heap double (roughly) every 2 heap size extensions. With 1.4 it's going to be a bit less than that.